static/js/urlparser.js

```diff
@@ -0,0 +1,9 @@
+function getUrlParam(key){
+    var urlParams = new URLSearchParams(window.location.search);
+    var param = urlParams.get(key);
+    if(param === null){
+        console.log("that url parameter key does not exist");
+        return;
+    }
+    return param;
+}
```

Comment:
The url parser is now in v2-beta, so it would be a good idea to rebase this branch again before changing the translations button to the entry in the options menu + modal.

To avoid conflicts I would recommend to do the following:
- Remove this file and the `<script src="static/js/urlparser.js"></script>` line from editor.html and commit
- Squash all the commits into 1
- Rebase this branch against v2-beta
- Implement the button to modal change